compiler/rustc_target/src/abi/call/mod.rs

```diff
@@ -658,7 +662,8 @@ impl<'a, Ty> FnAbi<'a, Ty> {
     {
         if abi == spec::abi::Abi::X86Interrupt {
             if let Some(arg) = self.args.first_mut() {
-                arg.make_indirect_byval();
+                // FIXME(pcwalton): This probably should use the x86 `byval` ABI...
```

Comment:
For `extern "x86-interrupt" fn(_: InterruptStackFrame)` the argument is passed as pointer without byval argument AFAIK.

Reply:
Are you saying I should change `make_indirect_byval()` to `make_indirect()` here?

Reply:
LLVM requires this argument to be marked `byval`. You'll get a verifier error otherwise.